lib/compat/wordpress-6.2/default-filters.php

```diff
@@ -20,3 +20,13 @@
 add_action( 'switch_theme', 'wp_theme_has_theme_json_clean_cache' );
 add_action( 'start_previewing_theme', 'wp_theme_has_theme_json_clean_cache' );
 add_action( 'upgrader_process_complete', '_wp_theme_has_theme_json_clean_cache_upon_upgrading_active_theme' );
+add_action( 'save_post_wp_global_styles', array( 'WP_Theme_JSON_Resolver_Gutenberg', 'clean_cached_data' ) );
+add_action( 'activated_plugin', array( 'WP_Theme_JSON_Resolver_Gutenberg', 'clean_cached_data' ) );
+add_action( 'deactivated_plugin', array( 'WP_Theme_JSON_Resolver_Gutenberg', 'clean_cached_data' ) );
+add_action( 'upgrader_process_complete', array( 'WP_Theme_JSON_Resolver_Gutenberg', '_clean_cached_data_upon_upgrading' ) );
+add_action( 'save_post_wp_global_styles', 'gutenberg_get_global_stylesheet_clean_cache' );
+add_action( 'switch_theme', 'gutenberg_get_global_stylesheet_clean_cache' );
+add_action( 'start_previewing_theme', 'gutenberg_get_global_stylesheet_clean_cache' );
+add_action( 'activated_plugin', 'gutenberg_get_global_stylesheet_clean_cache' );
+add_action( 'deactivated_plugin', 'gutenberg_get_global_stylesheet_clean_cache' );
+add_action( 'upgrader_process_complete', '_gutenberg_get_global_stylesheet_clean_cache_upon_upgrading' );
```

Comment:
```suggestion
add_action( 'upgrader_process_complete', '_gutenberg_get_global_stylesheet_clean_cache_upon_upgrading', 10, 2 );
```

Trying to reproduce locally, but without telling the hook it passes 2 parameters, it'll create a PHP fatal.

Reply:
Instructions to reproduce at https://github.com/WordPress/gutenberg/pull/45881